module/migration.mjs

```diff
@@ -843,6 +848,30 @@ function _migrateEffectArmorClass(effect, updateData) {
 
 /* -------------------------------------------- */
 
+/**
+ * Change active effects that target movement or senses.
+ * @param {object} effect      Effect data to migrate.
+ * @param {object} updateData  Existing update to expand upon.
+ * @returns {object}           The updateData to apply.
+ */
+function _migrateEffectMovementSenses(effect, updateData) {
+  let containsUpdates = false;
+  const changes = (effect.changes || []).map(c => {
+    if ((c.key.startsWith("system.attributes.movement.") || c.key.startsWith("system.attributes.senses."))
```

Comment:
This looks like it will move any changes targeting `attributes.movement.units`, `attributes.movement.hover`, or `attributes.senses.special` into the types object.